### ISSUE TITLE
GitHub teams for signup and authentication in test runner

### ISSUE THREAD

**Issue by zcorpan** [w3c/aria-at#167]
We're planning to use a github teams and github authentication as the signup and login flow for running aria-at tests in the new runner. For now, we can manually add members in GitHub's UI.

There are two aria-at teams in the w3c org: `aria-at` and `aria-at-editors`. We could use these teams for testers and admins in the test runner, I think. Or is there a reason it would be good to add new teams specifically for the runner?

I would like to have "maintainer" permissions for at least the team we decide to use for testers, so I can invite users to that team. The [pilot test](https://github.com/w3c/aria-at/issues/162) and usability tests are planned to take place the week of May 11.

**Comment by zcorpan** [comment]
@jnurthen and @mcking65 are maintainers currently in the [aria-at team](https://github.com/orgs/w3c/teams/aria-at/members). Can you add me as a maintainer as well?

**Comment by michael-n-cooper** [comment]
I presumed Matt's consent and made Simon a maintainer of the aria-at team. If you find a need to set up additional teams just let me know.

**Comment by zcorpan** [comment]
Thanks!

**Comment by zcorpan** [comment]
@michael-n-cooper So, as a stop-gap, we used teams in Bocoup org, but we should have the teams in the w3c org. I now think the teams for the app should be separate to the current `aria-at` and `aria-at-editors` teams for clarify. We also have both production and staging for the app with different teams.

Please create these teams in the w3c org:

* `aria-at-app testers`
* `aria-at-app admin`
* `aria-at-app staging testers`
* `aria-at-app staging admin`

Please add the following people with Maintainer role to all those teams:

* @zcorpan
* @mcking65 
* @mfairchild365 
* @spectranaut 
* @evmiguel
* @s3ththompson 

(or only add me and I can add the others)

Time sensitivity: ASAP, but Sunday this week at the latest. Apologies for late notice.

Thank you!

**Comment by michael-n-cooper** [comment]
Got those four teams set up, and all those people added to each team with maintainer access. In addition:

* The "aria-at-app admin" team has maintainer access to the repository.
* The "aria-at-app admin" and "aria-at-app staging admin" teams have write access to the master branch.
* The "aria-at-app admin" team has write access to the gh-pages branch.

Let me know if you need any of those settings tweaked.

**Comment by zcorpan** [comment]
They don't need any particular GitHub access, but it's OK for the "aria-at-app admin" team to have it.

Thank you!

**Comment by alflennik** [comment]
@michael-n-cooper Hi Michael, I'm the latest member of Bocoup's ARIA-AT team 😀

Could you add me as member of the w3c organization and invite me to the ARIA-AT team?

**Comment by alflennik** [comment]
@michael-n-cooper Hi Michael, could you add @aleenaloves as a member of the w3c organization? She will be joining the ARIA-AT team 🙂

Also, could you add me as a maintainer on the [ARIA-AT team](https://github.com/orgs/w3c/teams/aria-at/members)? Bocoup's current maintainers, Seth and Simon, have both moved onto other opportunities and we'd like to keep the ability to add and remove members to the team! Thanks!

**Comment by michael-n-cooper** [comment]
Hi @alflennik I've made you a maintainer and invited @aleenaloves to the team.

**Comment by alflennik** [comment]
Hi @michael-n-cooper our team is growing: could you add @paul-clue and @ccanash to the W3C org and the github team? Thank you!

**Comment by michael-n-cooper** [comment]
Hi @alflennik I've sent @Paul-Clue and @ccanash invites to the aria-at team.
